### DISCUSSION TITLE
Commits not being included in Contribution Activity

### DISCUSSION THREAD

**Discussion by nuzzles** [community/community#21571] [New to GitHub]
<p>Hello,</p>
<p>I have an issue where GitHub is not counting my contribution activity for a reason unknown to me.</p>
<p>My profile: <a href="https://github.com/simbleau" class="inline-onebox" rel="noopener nofollow ugc">simbleau (Spencer Imbleau) · GitHub</a><br>
From June 6th - 10th, I see no contribution activity.<br>
Although during this period, I was committing daily here: <a href="https://github.com/simbleau/convo/commits/dev" class="inline-onebox" rel="noopener nofollow ugc">Commits · simbleau/convo · GitHub</a></p>
<p>Can someone explain to me why the contributions do not count?</p>
<p>Thank you.<br>
~simbleau</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>At a glance, those commits don’t seem to be part of the default branch of that repository yet, which is required for them to count. They should appear after you merge.</p>
<p>See also: <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/managing-contribution-graphs-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile" class="inline-onebox">Why are my contributions not showing up on my profile? - GitHub Docs</a></p>

**Comment by nuzzles** [comment]
<p>If I do a squash-and-merge, will it only as 1 contribution (on the day of the squash), or would I have to merge the branch without squad to receive retroactive contribution?</p>

**Comment by airtower-luna** [comment]
<p>If you squash there will be only the one squash commit in your contribution history.</p>
